python/graphscope/nx/tests/test_ctx_builtin.py

```diff
@@ -250,3 +250,65 @@ def test_pagerank(self):
     def test_degree_assortativity_coefficient(self):
         ans = nx.builtin.degree_assortativity_coefficient(self.p2p_undirected)
         assert almost_equal(ans, -0.0925578, places=5)
+
+    def test_average_degree_connectivity(self):
```

Comment:
we should also add a test case to test graphscope graph.  attribute and degree app need this too